### DISCUSSION TITLE
how to updata the bib file in vscode? when i change the bib file, typst don't follow me.

### DISCUSSION THREAD

**Discussion by clq9920** [typst/typst#1718] [Q&A] [answered]
how to updata the bib file in vscode? when i change the bib file, typst don't follow me.

**Comment by laurmaedje** [comment]
Can you give a bit more details here? What do you mean with "typst don't follow me"?

**Reply by clq9920** [reply]
When I add a new literature to the bib file and reference it in the typ file, vscode prompts me that I cannot find it. The vscode plugin I use implements real-time preview for Typst Preview and Typst LSP. After several updates, this issue seems to have been resolved, and I am not sure if it will still occur.

**Reply by laurmaedje** [reply, marked as answer]
Might have been because Typst LSP isn't/wasn't aware of which file is the main one and then didn't know about the bibliography.
